services/shhext/chat/sql_lite_persistence.go

```diff
@@ -51,6 +53,63 @@ func NewSQLLitePersistence(path string, key string) (*SQLLitePersistence, error)
 	return s, nil
 }
 
+func MigrateDBFile(oldPath string, newPath string, key string) error {
+	_, err := os.Stat(oldPath)
+
+	// No files, nothing to do
+	if os.IsNotExist(err) {
+		return nil
+	}
+
+	// Any other error, throws
+	if err != nil {
+		return err
+	}
+
+	if err = os.Rename(oldPath, newPath); err != nil {
+		return err
+	}
+
+	db, err := openDB(newPath, "ON")
```

Comment:
Some explanation would be nice how this migration works. We use `"ON"` key here which is used in `PRAGMA key=%s` call and later we do `"PRAGMA rekey=%s"`.

Reply:
We used `ON` before as a key, so we could retrieve the db data for debugging purposes, now that we go live we actually re-key using the user password, I can add a comment